package.json

```diff
@@ -26,8 +26,8 @@
     "build:cjs": "BABEL_ENV=cjs babel src --extensions '.js,.ts,.tsx' --out-dir cjs/ --ignore 'src/index.es.js','**/__tests__','**/__mocks__' --quiet",
     "build:es": "BABEL_ENV=es babel src --extensions '.js,.ts,.tsx' --out-dir es/ --ignore 'src/index.es.js','**/__tests__','**/__mocks__' --quiet && BABEL_ENV=es babel src/index.es.js --out-file es/index.js --quiet",
     "doctoc": "doctoc --no-title --maxlevel 3 README.md CONTRIBUTING.md",
-    "storybook": "start-storybook -p 6006 --ci --static-dir .storybook/static",
-    "storybook:build": "build-storybook --output-dir website/stories --static-dir .storybook/static",
+    "storybook": "start-storybook --quiet --port 6006 --ci --static-dir .storybook/static",
```

Comment:
I added quiet modes to reduce the unnecessary pollution in the terminal. This option in the `storybook:build` script should also speed up the build time on Netlify.